webpack.polyfills.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = () => {
+  const config = {
+    mode: 'production',
+    entry: {
+      'intersection-observer-polyfill':
+        './libs/designsystem/src/lib/polyfills/intersection-observer-polyfill.js',
+      'intersection-observer-polyfill-loader':
+        './libs/designsystem/src/lib/polyfills/intersection-observer-polyfill-loader',
+    },
+    output: {
+      filename: '[name].min.js',
+      path: path.resolve(__dirname, 'libs/designsystem/src/lib/polyfills/'),
+    },
+    plugins: [
+      new CopyWebpackPlugin({
+        patterns: [
+          {
+            from: './libs/designsystem/src/lib/polyfills/intersection-observer-polyfill-loader.js',
+          },
+        ],
+      }),
+    ],
+  };
+
+  return config;
+};
```

Comment:
I also think that we can remove this file, since the polyfills have been removed.

Reply:
Would probably be a good idea to go through these two PRs and verify that deleted files are still deleted in this PR:
https://github.com/kirbydesign/designsystem/pull/2700/files
https://github.com/kirbydesign/designsystem/pull/2701/files